src/device/mod.rs

```diff
@@ -2678,7 +2691,7 @@ mod tests {
 
         rt.test_env
             .adapter
-            .expect_send_uapi_cmd_generic_call(1)
+            .expect_send_uapi_cmd_generic_call(2)
```

Comment:
I feel you should have been able to reuse singualar uapi call

Reply:
Leftover from the PoC. Cleaned it up.